KEYS

```diff
@@ -149,4 +149,62 @@ qdCDQHoUOgtZ4v6+QaxI3WV1KPBsPb7SAjuphubIQVK/6qHse9OoWVwWAABXHFqX
 +YMbPW0/IQ==
 =g11H
 -----END PGP PUBLIC KEY BLOCK-----
+pub   rsa4096 2019-05-15 [SC]
```

Comment:
Small nit... Can you add a blank line between `-----END PGP PUBLIC KEY BLOCK-----` and the start of your content

Reply:
Can do and thought about this. One thing though - would this blank line be added if you used the instruction from KEYS file per verbatim?

Reply:
Added the blank line

Reply:
> would this blank line be added if you used the instruction from KEYS file per verbatim?

@hickeyma I'm not entirely sure. Would need to dig more into this.

Reply:
That's ok. just asking.